packages/core/contracts/cross-chain-oracle/chain-adapters/Arbitrum_ParentMessenger.sol

```diff
@@ -34,8 +34,6 @@ contract Arbitrum_ParentMessenger is
     );
     event MessageReceivedFromChild(bytes data, address indexed childMessenger, address indexed targetHub);
 
-    // TODO: Can these default values be determined dynamically via L1 Arbitrum system contracts?
```

Comment:
it turns out these can only really be estimated deterministically from the L2 side